server/test/repository/AccountRepositoryTest.java

```diff
@@ -350,9 +361,16 @@ public void deleteUnusedGuestAccounts() {
 
   @Test
   public void addIdTokenAndPrune() {
+    when(mockSettingsManifest.getSessionReplayProtectionEnabled()).thenReturn(false);
     AccountModel account = new AccountModel();
     String fakeEmail = "fake email";
     account.setEmailAddress(fakeEmail);
+    // Clock that is within the default session duration
+    Clock clock = Clock.fixed(Instant.now().minusSeconds(10000), ZoneId.systemDefault());
```

Comment:
(blocking) It's a bit hard to follow what makes the time valid/invalid without knowing the duration. It would increase the readability if instead of `clock` calling it `validSessionClock` or something like that. here and on the other clocks.

Reply:
Good call, updated!